flaml/autogen/agent/agent.py

```diff
@@ -30,19 +31,49 @@ def _remember(self, memory):
         """Remember something."""
         self._memory.append(memory)
 
-    def _send(self, message, recipient):
-        """Send a message to another agent."""
-        self._conversations[recipient.name].append({"content": message, "role": "assistant"})
+    def _send(self, message: dict, recipient: str):
+        """Send a message to another agent.
+
+        An agent always assumes itself as the "assistant", and the message it receives is from the "user`".
```

Comment:
Is this right?
One important question to discuss is what is "message".

Reply:
Got it!